test/battle/move_effect/coaching.c

```diff
@@ -1,8 +1,112 @@
 #include "global.h"
 #include "test/battle.h"
 
```

Comment:
```suggestion

ASSUMPTIONS
{
    ASSUME(gMovesInfo[MOVE_COACHING].effect == EFFECT_COACHING);
}

```
Should have the assumption for Coaching effect.